REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.5
 Compat 0.19.0
 Reexport
+StatsBase
```

Comment:
Should require the latest version of StatsBase now that it's been tagged. Otherwise, should be good to go.